src/decision-engine/task-merger.js

```diff
@@ -0,0 +1,75 @@
+'use strict'
+
+const TaskMerger = {
+  /**
+   * Indicates whether the given task has newer information than the active
+   * tasks with the same topic
+   * @param {Task} task
+   * @param {Task[]} tasksWithTopic
+   * @returns {boolean}
+   */
+  hasNewInfo (task, tasksWithTopic) {
+    let haveSize = false
```

Comment:
maybe this should be `haveBlock`?  `isWantBlock` was easier to follow as it is consistent for both existing data and the task data.